connext_cmake_module/cmake/Modules/FindConnext.cmake

```diff
@@ -217,14 +216,15 @@ if(NOT "${_NDDSHOME} " STREQUAL " ")
   set(Connext_FOUND TRUE)
 else()
   # try to find_package() it
-  find_package(ndds_cpp)
-  if(ndds_cpp_FOUND)
-    message(STATUS "Found RTI Connext: ${ndds_cpp_DIR}")
-    set(Connext_INCLUDE_DIRS ${ndds_cpp_INCLUDE_DIRS})
-    set(Connext_LIBRARIES ${ndds_cpp_LIBRARIES})
+  find_package(nddscpp)
+  find_package(rticonnextmsgcpp)
+  if(nddscpp_FOUND AND rticonnextmsgcpp_FOUND)
+    message(STATUS "Found RTI Connext: ${nddscpp_DIR}")
+    set(Connext_INCLUDE_DIRS ${nddscpp_INCLUDE_DIRS})
+    set(Connext_LIBRARIES ${rticonnextmsgcpp_LIBRARIES})
```

Comment:
The PR can be merged as-is but this should be cleaned up sometime in the near future (if we don't a Connext version which just solves that in a single package).